### PR TITLE
feat(styles-migration): add automatic migration for bootstrap form-control, form-select and textarea components

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

**Comment by oliverschuerch** [comment]
Review #687 first. This will make it easier to review this PR.

**Comment by gfellerph** [comment]
@oliverschuerch can you target that PR at #687? This way we can merge the whole feature when it's complete and it's easier to review this PR.

File: .changeset/spicy-boats-judge.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-demo': patch
+---
+
+Upated migration documentation for bootstrap `form-control`, `form-select` and `textarea` components.
+Removed floating-label-wrapper and form-control-lg-class changes from `textarea` and `form-file` components, because they never existed.
```
Comment (gfellerph) [inline .changeset/spicy-boats-judge.md:6]: Is this something that changed from v4 to v5 or just in v5 itself? I think it should be elaborated a little to make sense to end-users.

**Review by gfellerph — APPROVED** [review]

File: .changeset/spicy-boats-judge.md (hunk 1 above)
Comment (oliverschuerch) [reply .changeset/spicy-boats-judge.md:6]: In v4 `textarea` and `form-file` components did not work with floating-labels or at least, there are no examples shown in the docs. Therefore these two components must have gotten a `floating-label` visuality only in v5.
We should just remove the second sentence, so we're fine.

**Pull request merged**